install-dev/data/db_structure.sql

```diff
@@ -1,2818 +1,2830 @@
-  `id_shop` int(10) unsigned NOT NULL, 
+  `id_product` int(10) unsigned NOT NULL,
+  `id_carrier_reference` int(10) unsigned NOT NULL,
+  `id_shop` int(10) unsigned NOT NULL,
   PRIMARY KEY (
-    `id_product`, `id_carrier_reference`, 
+    `id_product`, `id_carrier_reference`,
     `id_shop`
   )
 ) ENGINE=ENGINE_TYPE DEFAULT CHARSET=utf8 COLLATION;
 
 /* Stats from guest user */
 CREATE TABLE `PREFIX_guest` (
-  `id_guest` int(10) unsigned NOT NULL auto_increment, 
-  `id_operating_system` int(10) unsigned DEFAULT NULL, 
-  `id_web_browser` int(10) unsigned DEFAULT NULL, 
-  `id_customer` int(10) unsigned DEFAULT NULL, 
-  `javascript` tinyint(1) DEFAULT '0', 
-  `screen_resolution_x` smallint(5) unsigned DEFAULT NULL, 
-  `screen_resolution_y` smallint(5) unsigned DEFAULT NULL, 
-  `screen_color` tinyint(3) unsigned DEFAULT NULL, 
-  `sun_java` tinyint(1) DEFAULT NULL, 
-  `adobe_flash` tinyint(1) DEFAULT NULL, 
-  `adobe_director` tinyint(1) DEFAULT NULL, 
-  `apple_quicktime` tinyint(1) DEFAULT NULL, 
-  `real_player` tinyint(1) DEFAULT NULL, 
-  `windows_media` tinyint(1) DEFAULT NULL, 
-  `accept_language` varchar(8) DEFAULT NULL, 
-  `mobile_theme` tinyint(1) NOT NULL DEFAULT '0', 
-  PRIMARY KEY (`id_guest`), 
-  KEY `id_customer` (`id_customer`), 
-  KEY `id_operating_system` (`id_operating_system`), 
+  `id_guest` int(10) unsigned NOT NULL auto_increment,
+  `id_operating_system` int(10) unsigned DEFAULT NULL,
+  `id_web_browser` int(10) unsigned DEFAULT NULL,
+  `id_customer` int(10) unsigned DEFAULT NULL,
+  `javascript` tinyint(1) DEFAULT '0',
+  `screen_resolution_x` smallint(5) unsigned DEFAULT NULL,
+  `screen_resolution_y` smallint(5) unsigned DEFAULT NULL,
+  `screen_color` tinyint(3) unsigned DEFAULT NULL,
+  `sun_java` tinyint(1) DEFAULT NULL,
+  `adobe_flash` tinyint(1) DEFAULT NULL,
+  `adobe_director` tinyint(1) DEFAULT NULL,
+  `apple_quicktime` tinyint(1) DEFAULT NULL,
+  `real_player` tinyint(1) DEFAULT NULL,
+  `windows_media` tinyint(1) DEFAULT NULL,
+  `accept_language` varchar(8) DEFAULT NULL,
+  `mobile_theme` tinyint(1) NOT NULL DEFAULT '0',
+  PRIMARY KEY (`id_guest`),
+  KEY `id_customer` (`id_customer`),
+  KEY `id_operating_system` (`id_operating_system`),
   KEY `id_web_browser` (`id_web_browser`)
 ) ENGINE=ENGINE_TYPE DEFAULT CHARSET=utf8 COLLATION;
 
 /* Store hook description */
 CREATE TABLE `PREFIX_hook` (
-  `id_hook` int(10) unsigned NOT NULL auto_increment, 
-  `name` varchar(64) NOT NULL, 
-  `title` varchar(64) NOT NULL, 
-  `description` text, 
-  `position` tinyint(1) NOT NULL DEFAULT '1', 
-  PRIMARY KEY (`id_hook`), 
+  `id_hook` int(10) unsigned NOT NULL auto_increment,
+  `name` varchar(64) NOT NULL,
+  `title` varchar(64) NOT NULL,
+  `description` text,
+  `position` tinyint(1) NOT NULL DEFAULT '1',
+  PRIMARY KEY (`id_hook`),
   UNIQUE KEY `hook_name` (`name`)
 ) ENGINE=ENGINE_TYPE DEFAULT CHARSET=utf8 COLLATION;
 
 /* Hook alias name */
 CREATE T
```

Comment:
Regarding the changes done in the ObjectModel Currency, is this table still needed?

Reply:
no idea